randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2847,7 +2850,9 @@ Extra - asset_id: collision_camera_034
   * Extra - actor_layer: breakables
   * Extra - tile_types: ('WEIGHT',)
   > Door to Wave Beam Tutorial
-      Gravity Suit or After Ferenia - Wave Emmi Lower Water
+      Any of the following:
+          Gravity Suit or After Ferenia - Wave Emmi Lower Water
+          Lay Cross Comb and Perform WBJ
```

Comment:
i believe you can change the Lay Cross Bomb to Simple IBJ. Should be able to IBJ out of the water with normal bombs too

Reply:
No it´s not possible with normal bombs.